internal/dvcore/run.go

```diff
@@ -198,6 +200,11 @@ func pullBatches(targets []multipmuri.Entity, h *cayley.Handle, githubToken stri
 
 				githubprovider.FetchRepo(ctx, repo, githubToken, out, ghOpts)
 			}(target)
+			case multipmuri.TrelloProvider:
+				go func(board multipmuri.Entity) {
+					defer wg.Done()
+					trelloprovider.FetchCard(ctx, board, trelloToken, trelloApiKey, target.LocalID()[1:], out)
```

Comment:
you should check `target.LocalID()` before calling `[1:]` in case it's empty.

I think it deserves a dedicated helper.